build.mill

```diff
@@ -317,7 +317,7 @@ def millBinPlatform: T[String] = Task {
   //  val pos = if (tag.startsWith("0.")) 2 else 1
   //  tag.split("[.]", pos + 1).take(pos).mkString(".")
   // }
-  "0.11"
+  "0.13"
```

Comment:
Instead of hardcoding, removing the hardcoded version and uncommenting the logic based on `millLastTag` seems what you want.